lib/fittings/swagger_render.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = function create(fittingDef, bagpipes) {
+    return function swagger_render(context, next) {
```

Comment:
Identifier 'swagger_render' is not in camel case.<br>'next' is defined but never used.